server/core/app.ts

```diff
@@ -26,6 +27,16 @@ export const returnExpress: (
   // add pino-http middleware
   app.use(generateHttpLogger());
 
+  //Add session middleware
+  const sessionOpts: SessionOptions = {
```

Comment:
I assume (in later issues) this will be enabled/added only when needed?

Reply:
I think it's fair to always have a session as we might want to store information for the lifetime of a browser session. In particular, for logging - @ajborley, would it be useful to store a "client id" in the session?